Packs/SophosCentral/Integrations/SophosCentral/SophosCentral.py

```diff
@@ -0,0 +1,1764 @@
+        for item in items:
+            current_object_data = create_detected_exploit_output(item)
+            outputs.append(current_object_data)
+    readable_output = f'### Current page: {int(args.get("page", ""))}\n'
+    readable_output += tableToMarkdown(name=f'Listed {len(outputs)} Detected Exploits:', t=outputs,
+                                       headers=table_headers, removeNull=True)
+    return CommandResults(outputs_key_field='id', outputs_prefix='SophosCentral.DetectedExploit',
+                          raw_response=results, outputs=outputs, readable_output=readable_output)
+
+
+def sophos_central_detected_exploit_get_command(client: Client, args: dict) -> CommandResults:
+    """
+    Get a single detected exploit.
+
+    Args:
+        client (Client): Sophos Central API client.
+        args (dict): All command arguments
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    result = client.get_detected_exploit(args.get('detected_exploit_id', ''))
+    table_headers = ['id', 'description', 'thumbprint', 'count', 'firstSeenAt', 'lastSeenAt']
+    object_data = {}
+    if result:
+        object_data = create_detected_exploit_output(result)
+
+    readable_output = tableToMarkdown(name='Found Detected Exploit:', t=object_data,
+                                      headers=table_headers, removeNull=True)
+    return CommandResults(outputs_key_field='id', outputs_prefix='SophosCentral.DetectedExploit',
+                          raw_response=result, outputs=object_data, readable_output=readable_output)
+
+
+def fetch_incidents(client: Client, last_run: Dict[str, int],
+                    first_fetch_time: str, fetch_severity: Optional[List[str]],
+                    fetch_category: Optional[List[str]],
+                    max_fetch: Optional[str]) -> Tuple[Dict[str, int], List[dict]]:
+    """
+    Fetch incidents (alerts) each minute (by default).
+    Args:
+        client (Client): Sophos Central Client.
+        last_run (dict): Dict with last_fetch object,
+                                  saving the last fetch time(in millisecond timestamp).
+        first_fetch_time (dict): Dict with first fetch time in str (ex: 3 days ago).
+        fetch_severity (list(str)): Severity to fetch.
+        fetch_category (list(str)): Category(s) to fetch.
+        max_fetch (str): Max number of alerts to fetch.
+    Returns:
+        Tuple of next_run (millisecond timestamp) and the incidents list
+    """
+    last_fetch_timestamp = last_run.get('last_fetch', None)
+    fetch_category = None if not fetch_category else fetch_category
+    fetch_severity = None if not fetch_severity else fetch_severity
+    limit = None if not max_fetch else int(max_fetch)
+
+    if last_fetch_timestamp:
+        last_fetch_date = datetime.fromtimestamp(last_fetch_timestamp / 1000)
+        last_fetch = last_fetch_date
+    else:
+        first_fetch_date = dateparser.parse(first_fetch_time)
```

Comment:
We always regard the incidents in demisto in utc
```suggestion
        first_fetch_date = dateparser.parse(first_fetch_time).replace(tzinfo=None)
```